### ISSUE TITLE
missing gym info

### ISSUE THREAD

**Issue by AlicanGultekin** [Armax/Pokemon-GO-node-api#183]
this is what we get now:
{   FortId: 'dummyvalue',
    LastModifiedMs: Long { low: 744978334, high: 342, unsigned: false },
    Latitude: somelat,
    Longitude: somelon,
    Team: 2,
    GuardPokemonId: 18,
    GuardPokemonLevel: null,
    Enabled: true,
    FortType: null,
    GymPoints: Long { low: 6194, high: 0, unsigned: false },
    IsInBattle: null,
    ActiveFortModifier: null,
    LureInfo: null,
    CooldownCompleteMs: null,
    Sponsor: null,
    RenderingType: null }

GuardPokemonLevel is always null and shouldn't there be more info coming in like who's the leading trainer and maybe the other traniners also?

**Comment by firanto** [comment]
Nope. That's the actual data that also received by the official game. The game only need to know what kind of pokemon guarding it, to shown it at the gym's top.

As for the data you are looking for (every guards on the gym, it's CP, who own it, also gym related info like name and picture) should be on another api call. You should notice this... When tapping gym to see it's details, there was a activity indicator spinning on the top left indicating api call is in progress. I haven't try it yet since my bot doesn't need it. But you could try calling Pokeio.GetFortDetails().

**Comment by AlicanGultekin** [comment]
i see, that makes sense.
i wasn't aware there's a separate function for it, thank you.

edit: 
this is what is returned; fort_id, name, image_urls, latitude, and longitude are correct but the rest of the data are just null.
{
    "fort_id": "somefort",
    "team_color": null,
    "pokemon_data": null,
    "name": "somefortsname",
    "image_urls": [
        "someimageurl"
    ],
    "fp": null,
    "stamina": null,
    "max_stamina": null,
    "type": null,
    "latitude": somelat,
    "longitude": somelon,
    "description": null,
    "modifiers": []
}
